server/lib/s3backup/__init__.py

```diff
@@ -0,0 +1,176 @@
+"""
+Simple module level convenience functions.
+"""
+import boto3
+import os
+import sys
+import glob
+import base64
+import hashlib
+import shutil
+import time
+import configtools
+from datetime import datetime
+from configparser import ConfigParser
+
+class Entry(object):
+    """
+    A simple class to create an entry as an object.
```

Comment:
"An Entry object consists of a name (of an S3 object) and the MD5 value of that object. It is used by ..." Basically: why do we need the Entry class?